scripts/import_cldr.py

```diff
@@ -286,15 +287,20 @@ def parse_global(srcdir, sup):
         region_code = region.attrib['iso3166']
         region_currencies = []
         for currency in region.findall('./currency'):
+            cur_code = currency.attrib['iso4217']
             cur_start = _parse_currency_date(currency.attrib.get('from'))
             cur_end = _parse_currency_date(currency.attrib.get('to'))
-            region_currencies.append((currency.attrib['iso4217'],
-                                      cur_start, cur_end,
-                                      currency.attrib.get(
-                                          'tender', 'true') == 'true'))
+            cur_tender = currency.attrib.get('tender', 'true') == 'true'
+            # Tie region to currency.
+            region_currencies.append((cur_code, cur_start, cur_end, cur_tender))
+            # Keep a reverse index of currencies to territorie.
+            all_currencies.setdefault(cur_code, []).append(region_code)
         region_currencies.sort(key=_currency_sort_key)
         territory_currencies[region_code] = region_currencies
 
+    # Deduplicate regions and makes them pickable.
+    all_currencies = dict([(code, tuple(set(regions))) for code, regions in all_currencies.items()])
```

Comment:
Um, this doesn't actually affect the `all_currencies` object that is already in `global_data`.
What do you mean with "pickable" (i.e. what is the requirement for the value to be a `tuple`)?

I mean, if the value was a `set` all the time, the deduplication code would be unnecessary:

```python
for currency in region.findall('./currency'):
    all_currencies.setdefault(cur_code, set()).add(region_code)
```